frontend/lib/about/license_page.dart

```diff
@@ -51,15 +53,15 @@ class CustomLicensePage extends StatelessWidget {
 
           return CustomScrollView(
             slivers: <Widget>[
-              const CustomSliverAppBar(title: 'Lizenzen'),
+              CustomSliverAppBar(title: context.l10n.about_licenses),
```

Comment:
I guess you removed the constant constructor as the value can now change?

Reply:
Yes, exactly.